build-system/pr-check/bundle-size.js

```diff
@@ -19,28 +19,72 @@
  * @fileoverview Script that runs the bundle-size checks during CI.
  */
 
+const {cyan, green, red} = require('../common/colors');
+const {execScriptAsync} = require('../common/exec');
+const {log} = require('../common/logging');
 const {runCiJob} = require('./ci-job');
 const {skipDependentJobs, timedExecOrDie} = require('./utils');
 const {Targets, buildTargetsInclude} = require('./build-targets');
 
 const jobName = 'bundle-size.js';
 
+/**
+ * Builds both Module and Nomodule builds of AMP in parallel.
+ * @return {Promise<void>}
+ */
+async function buildModuleNomodule() {
+  const buildPromises = [];
+  ['--esm', '--no-esm'].forEach((esmFlag) => {
```

Comment:
If this works, something is wrong with the parallelism of our build step.

The underlying implementation attempts to use as many cores as provided.

Reply:
This is a draft, I'm just checking the limitations of CircleCI :D